map-of-the-web/motw/motw/producing.py

```diff
@@ -5,11 +5,10 @@
 import requests as req
 from datetime import datetime
 import pandas as pd
+from util import *
 
-def timestamp2date(x):
```

Comment:
This change (and the incorporation of util) should be put in the commit where you wrote the utility function.  Note, you can add individual changes from a file by using git add -p